packages/documentation/docs/installation/angular.md

```diff
@@ -3,11 +3,16 @@ sidebar_position: 1
 sidebar_title: Angular
 title: Angular
 ---
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ### Library installation
 
+:::info
+Currently, Angular standalone components are not supported by iX. We are actively working to provide support for them soon. Otherwise Siemens iX is fully compatible with Angular 17 and later versions.
```

Comment:
```suggestion
Currently, iX only supports `NgModule` based Angular projects. We are actively working to provide support for the new standalone component based approach soon. Otherwise Siemens iX is fully compatible with Angular version 17 and higher.
```